en/includes/guides/branding/add-custom-layouts.md

```diff
@@ -0,0 +1,144 @@
+# Customize layouts
+
+You can customize the layout and branding of user interfaces like the login page, registration page, username and password recovery pages, and single sign-on pages.
+
+All UIs are available in the following apps located in the `<IS_HOME>/repository/deployment/server/webapps` folder.
+
+- `authenticationendpoint`
+- `accountrecoveryendpoint`
+ - `x509certificateauthenticationendpoint`
+
+## General components of the UI
+
+All pages of WSO2 Identity Server are separated into three general components as shown below. You can reference these components when you form the custom layout.
+
+![Login Page]({{base_path}}/assets/img/guides/branding/login-page-labelled.png)
+
+## Layout syntaxes
+
+This section explains the special syntax that you can use to create a custom layout.
+
+- **Condition block**
+    
+    This is similar to an `IF` block. In the following example, the condition block will activate or deactivate according to the value of the `isPolicyPage` variable.
+
+    ![Condition Block]({{base_path}}/assets/img/guides/branding/condition-block.png){: width="700"}
```

Comment:
All of these code examples need to be updated. Get code examples from https://github.com/wso2/identity-apps/blob/master/identity-apps-core/components/login-portal-layouts/layouts/centered/body.html